ansible/roles/helios_broker_setup/tasks/main.yml

```diff
@@ -0,0 +1,142 @@
+---
+  - debug: var=helios_dockerhub_org
+
+  - debug: var=helios_broker_template_dir
+
+  # Verify that the template file 'helios_asb_template_filename' exists
+  - stat:
+      path: "{{ helios_broker_template_dir }}/{{ helios_asb_template_filename }}"
```

Comment:
We will want to add ability to fetch the file, eventually it will exist in a location in S3

Reply:
will update